packages/main/src/Dialog.ts

```diff
@@ -210,7 +91,137 @@ const metadata = {
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	stretch!: boolean;
+
+	/**
+	 * Determines whether the component is draggable.
+	 * If this property is set to true, the Dialog will be draggable by its header.
+	 * <br><br>
+	 * <b>Note:</b> The component can be draggable only in desktop mode.
+	 * <br><br>
+	 * <b>Note:</b> This property overrides the default HTML "draggable" attribute native behavior.
+	 * When "draggable" is set to true, the native browser "draggable"
+	 * behavior is prevented and only the Dialog custom logic ("draggable by its header") works.
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.Dialog.prototype.draggable
+	 * @defaultvalue false
+	 * @since 1.0.0-rc.9
+	 * @public
+	 */
+	@property({ type: Boolean })
+	draggable!: boolean;
+
+	/**
+	 * Configures the component to be resizable.
+	 * If this property is set to true, the Dialog will have a resize handle in its bottom right corner in LTR languages.
+	 * In RTL languages, the resize handle will be placed in the bottom left corner.
+	 * <br><br>
+	 * <b>Note:</b> The component can be resizable only in desktop mode.
+	 * <br>
+	 * <b>Note:</b> Upon resizing, externally defined height and width styling will be ignored.
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.Dialog.prototype.resizable
+	 * @defaultvalue false
+	 * @since 1.0.0-rc.10
+	 * @public
+	 */
+	@property({ type: Boolean })
+	resizable!: boolean;
+
+	/**
+	 * Defines the state of the <code>Dialog</code>.
+	 * <br>
+	 * Available options are: <code>"None"</code> (by default), <code>"Success"</code>, <code>"Warning"</code>, <code>"Information"</code> and <code>"Error"</code>.
+	 * @type {sap.ui.webc.base.types.ValueState}
+	 * @name sap.ui.webc.main.Dialog.prototype.state
+	 * @defaultvalue "None"
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	@property({ type: ValueState, defaultValue: ValueState.None })
+	state!: ValueState;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	onPhone!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	onDesktop!: boolean;
+
+	_screenResizeHandler!: () => void;
+	_dragMouseMoveHandler!: (e: MouseEvent) => void;
+	_dragMouseUpHandler!: (e: MouseEvent) => void;
+	_resizeMouseMoveHandler!: (e: MouseEvent) => void;
+	_resizeMouseUpHandler!: (e: MouseEvent) => void;
+	_dragStartHandler!: (e: DragEvent) => void;
+	_y?: int;
+	_x?: int;
+	_isRTL?: boolean;
+	_screenResizeHandlerAttached?: boolean;
+	_initialX?: number;
+	_initialY?: number;
+	_initialWidth?: number;
+	_initialHeight?: number;
+	_initialTop?: number;
+	_initialLeft?: number;
+	_minWidth?: number;
+	_cachedMinHeight?: number;
+
+	/**
+	 * Defines the header HTML Element.
+	 * <br><br>
+	 * <b>Note:</b> If <code>header</code> slot is provided, the labelling of the dialog is a responsibility of the application developer.
+	 * <code>accessibleName</code> should be used.
+	 *
+	 * @type {HTMLElement[]}
+	 * @slot
+	 * @public
```

Comment:
name for the slots is also required

Reply:
I meant @name sap.ui.webc.main.Dialog.prototype.header
and
sap.ui.webc.main.Dialog.prototype.footer

sorry for not clarifying it

Reply:
Thanks, i fixed it now.